README.md

```diff
@@ -1,75 +1,16 @@
-```
-
-CLI for Node.js is available from within any directory by running `suitecloud`.
-
-## Usage
-CLI for Node.js uses the following syntax: 
-```
-suitecloud <command> <option> <argument>
-```
-
-### Commands
-| Command | Description |
-| --- | --- |
-| [`adddependencies`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_155981452469.html)| Adds missing dependencies to the manifest file.|
-|[`createproject`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156041348327.html)|Creates a SuiteCloud project, either a SuiteApp or an account customization project (ACP).|
-|[`deploy`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156044636320.html)|Deploys the folder containing the project.|
-|[`importfiles`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156041963273.html)|Imports files from an account to your account customization project.|
-|[`importobjects`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156042181820.html)|Imports custom objects from an account to your SuiteCloud project.|
-|[`listfiles`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156042966488.html)|Lists the files in the File Cabinet of your account.|
-|[`listobjects`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156043303237.html)|Lists the custom objects deployed in an account.|
-|[`localserver`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156043691482.html)|Generates a local server of your SuiteCommerce extensions and themes.|
-|[`manageauth`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_157304934116.html)|Manages authentication IDs (authid) for all your projects.|
-|[`proxy`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156044426239.html)|Configures a proxy server.|
-|[`setupaccount`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156044528841.html)|Sets up an account to use with the SuiteCloud CLI for Node.js.|
-|[`update`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156050566547.html)|Overwrite the custom objects in the project with the custom objects in an account.|
-|[`validate`](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_156049843194.html)|Validates the folder containing the SuiteCloud project.|
-
-To check the help for a specific command, run the following command:
-```
-suitecloud {command} -h
-```
-
-Read the detailed documentation for all the commands in [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=chapter_155931263126.html).
-
-## Getting Started
-Create a new project in an empty folder by running the following command:
-```
-suitecloud createproject -i
-```
-
-After you create a project, configure a NetSuite account, by running the following command within the project folder:
-```
-suitecloud setupaccount
-```
+You can also extend SuiteCloud CLI for Node.js with the following command:
```

Comment:
Yes, I just wanted to mention that this command is part of CLI for Node.js as a complement.

Reply:
yep

Reply:
Then, who would install the command by itself if it only works with the CLI for Node.js? Because if the CLI is a dependency, they will already have the command when they install the CLI.

Reply:
I deleted this mention about the localserver command.